.circleci/config.yml

```diff
@@ -26,6 +26,19 @@ executors:
     docker:
       - image: cimg/aws:2024.03
 commands:
+  install_cf_tools:
+    description: "Install Cloud Foundry CLI"
+    steps:
+      - run:
+          name: Install Git
```

Comment:
does this name represent what is being done in the job?

Reply:
![Screenshot 2024-12-19 at 9 47 04 AM](https://github.com/user-attachments/assets/2ddd9be9-6621-4451-b735-8f71476b096a)

yeah lets fix this

Reply:
i dont know where its doing it twice

Reply:
fount it